packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -103,17 +102,18 @@ class PaginatedDataTable extends StatefulWidget {
        assert(source != null),
        super(key: key);
 
-  /// The table card's header.
+  /// The table card's optional header.
   ///
   /// This is typically a [Text] widget, but can also be a [ButtonBar] with
```

Comment:
I'd prefer it if we'd leave the reference to ButtonBar (a problematic widget) out of the API doc. A [Row] of [TextButtons] would suffice here.

It would be useful to refer to [actions] here. Note also: it seems a little odd to include buttons or actions in the header if "when the selection is not empty, the header is replaced by a count of the selected items".

Reply:
Updated now!